src/main/java/rua/command/MarkCommand.java

```diff
@@ -0,0 +1,65 @@
+package rua.command;
+
+import rua.common.Storage;
+import rua.common.Ui;
+import rua.task.TaskList;
+
+public class MarkCommand implements Command {
+    private final Boolean marked;
```

Comment:
Should this be `isMarked`?
```suggestion
    private final Boolean isMarked;
```